tv_apps/smart-system/js/app_install_manager.js

```diff
@@ -590,6 +624,28 @@ var AppInstallManager = {
     var dialog = this.downloadCancelDialog;
     dialog.classList.remove('visible');
     delete dialog.dataset.manifest;
+  },
+
+  hookSimpleNavigator: function(navigableButtons, defaultFocusButton) {
+    var that = this;
+    this.simpleKeyNavigation.start(navigableButtons,
+      SimpleKeyNavigation.DIRECTION.HORIZONTAL);
+    window.setTimeout(function() {
+      if (document.activeElement) {
+        document.activeElement.blur();
+      }
+      if (defaultFocusButton) {
+        that.simpleKeyNavigation.focusOn(that.defaultFocusButton);
```

Comment:
What's `that.defaultFocusButton`? I think this is typo. It should be `defaultFocusButton`.